lib/datalifecycle/purge/policy.go

```diff
@@ -0,0 +1,62 @@
+package purge
+
+import (
+	"context"
+	"time"
+
+	es "github.com/chef/automate/api/interservice/es_sidecar"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+// Policy represents a purge workflow purge policy
+type Policy struct {
```

Comment:
Maybe make this `ElasticsearchPolicy` so we can more easily add `PostgresPolicy`'s for the application-service.